PowerShell/ScubaGear/Modules/Providers/ExportAADProvider.psm1

```diff
@@ -332,7 +388,12 @@ function Get-PrivilegedUser {
                     }
 
                     # Get the users that are assigned to the PIM group as Eligible members
-                    $PIMGroupMembers = Get-MgBetaIdentityGovernancePrivilegedAccessGroupEligibilityScheduleInstance -All -ErrorAction Stop -Filter "groupId eq '$UserObjectId'"
+                    # $PIMGroupMembers = Get-MgBetaIdentityGovernancePrivilegedAccessGroupEligibilityScheduleInstance -All -ErrorAction Stop -Filter "groupId eq '$UserObjectId'"
```

Comment:
Is there a good reason to keep this commented out line? Otherwise recommend for deletion.
```suggestion
```

Reply:
I thought it might be good to keep these so people would understand what the `Invoke-GraphDirectly` function is doing, and easily revert them if issues arise but...that's probably not a good reason. I will nuke these.